adapter/service/parameters/model.go

```diff
@@ -0,0 +1,66 @@
+package parameters
+
+const (
+	WidgetTypeInput       = "input"
+	WidgetTypeSelect      = "select"
+	WidgetTypeMultiSelect = "multiselect"
+
+	ValueTypeInt         = "int"
+	ValueTypeString      = "string"
+	ValueTypeBool        = "bool"
+	ValueTypeIntArray    = "int_array"
+	ValueTypeStringArray = "string_array"
+)
+
+// TODO: add helper methods
+type SupportedParameter struct {
+	ID           string         `json:"parameter_id"`
+	Name         string         `json:"name"`
+	Description  string         `json:"description"`
+	ValueType    string         `json:"value_type"`
+	WidgetType   string         `json:"widget_type"`
+	Options      []SelectOption `json:"options,omitempty"`
+	Min          int            `json:"min,omitempty"`
+	Max          int            `json:"max,omitempty"`
+	DefaultValue int            `json:"default_value"`
```

Comment:
In spec this is ANY, so interface, or a custom union type with custon JSON serialization method.

Reply:
Yeah, IDE prank ;) Good catch! Fixed.